reconcile/openshift_base.py

```diff
@@ -219,13 +219,10 @@ def apply(dry_run, oc_map, cluster, namespace, resource_type, resource,
 
         try:
             oc.apply(namespace, annotated.toJSON())
-        except ApplyError as e:
-            if 'Invalid value: 0x0' in str(e):
-                oc.remove_last_applied_configuration(
-                    namespace, resource_type, resource.name)
-                oc.apply(namespace, annotated.toJSON())
-            else:
-                raise StatusCodeError(str(e))
```

Comment:
Did anything change at all with this pull request?
Before, you'd call `StatusCodeError` here.